common/src/model_registration/register.py

```diff
@@ -130,23 +135,57 @@ def main(args):
             model_version = str(int(max_version) + 1)
     except Exception:
         print(f"Error in listing versions for model {model_name}. Trying to register model with version '1'.")
+    
+    # Updating tags and properties with value provided in metadata file
+    if args.model_metadata_path:
+        with open(args.model_metadata_path, 'r') as stream:
+            metadata = yaml.safe_load(stream)
+            tags = metadata['tags']
```

Comment:
properties might or might not be present. Use `metadata.get("tags", tags)` and so forth